pkg/virtctl/imageupload/imageupload.go

```diff
@@ -347,60 +341,60 @@ func (c *command) run(args []string) error {
 		fmt.Printf("Using existing PVC %s/%s\n", namespace, pvc.Name)
 	}
 
-	if createPVC {
-		err = waitUploadServerReady(virtClient, namespace, name, uploadReadyWaitInterval, time.Duration(uploadPodWaitSecs)*time.Second)
+	if c.createPVC {
+		err = waitUploadServerReady(virtClient, namespace, c.name, 1*time.Second, time.Duration(c.uploadPodWaitSecs)*time.Second)
```

Comment:
Why remove `uploadReadyWaitInterval`? Also, you're replacing the value, before it was `2*time.Second`.

Reply:
I'll revert it. I had some weird timeout issue, that only with `1*time.Second` unit tests passed locally (and with this `uploadReadyWaitInterval` variable it failed).

Reply:
I just thought `uploadReadyWaitInterval` is not needed on the scope of the whole script, just in this case it can be where it's used.